### PR TITLE
various rockspec fixes

### PR DESCRIPTION
**Pull request by ghost**
Will tag it to 2.1.1 once approved and merged.

### CODE REVIEW

**Comment by mikefero** [comment]
Looks good, but we may also want to change the following:

```diff
-  homepage = "https://github.com/leafo/pgmoon",
-  maintainer = "Leaf Corcoran <leafot@gmail.com>"
+  homepage = "https://github.com/Kong/pgmoon",
+  maintainer = "KongHQ"
```

Not sure if the maintainer needs an e-mail address or not.

**Comment by mikefero** [comment]
Doesn't the rockspec filename need to match the version?

`pgmoon-kong.rockspec` => `pgmoon-kong-2.1.1-1.rockspec`

If the version number isn't required by `luarocks make` in the rockspec filename then that works for me.

File: pgmoon-kong-2.1.1-1.rockspec [hunk 1]
```diff
@@ -1,15 +1,16 @@
 package = "pgmoon"
-version = "dev-1"
+version = "2.1.1"
```
Comment (mikefero) [inline pgmoon-kong-2.1.1-1.rockspec]:
```suggestion
version = "2.1.1-1"
```

In order to get the `luarocks make` to be successful (and match the format expected) you will also need to change `pgmoon-kong-2.1.1-1.rockspec` => `pgmoon-2.1.1-1.rockspec`.

These two things should fix the build.

**Review by mikefero — CHANGES_REQUESTED** [review]

**Comment by ghost** [comment]
Remaining test failures seems to be unrelated to this PR since it has been happening for a while looking at the history: https://github.com/Kong/pgmoon/commits/feat/mtls

`ngx.ssl` is only available in openresty land, whereas this plugin's CI does not have it enabled/installed to run tests.

**Pull request closed without merging**